### PR TITLE
Direct links based on @helrond CRALS commit

### PR DESCRIPTION
**Pull request by kieranjol**
I know very little about web design, I just tried my best to mimic @helrond. https://github.com/dd388/crals/commit/7c614298789dbda49b3637c79c2ad0b6f0924519

### CODE REVIEW

**Comment by ablwr** [comment]
Kieran you are so great!!!!!!!!!!!!!!

**Comment by ablwr** [comment]
Looks fine on local server. Merging.

**Comment by kieranjol** [comment]
:+1: :smile:

**Comment by todrobbins** [comment]
Great work @kieranjol! You beat me to it. LOL

**Comment by retokromer** [comment]
@kieranjol Is there any reason for passing the class parameter
`<p class="link"></p>`
in such an... obfuscated way?

**Comment by kieranjol** [comment]
Hi Reto,

I don't know. Any improvement would be welcome, I just tried to adapt the
crals code until it worked.

**Comment by retokromer** [comment]
Thank you, @kieranjol! Then I'll move them into an already existing element. It's a little easier to maintain, in my opinion.

**Comment by kieranjol** [comment]
Thanks Reto, I look forward to seeing the changes. I know very little about
this stuff so it'll be good to see it improved.

**Comment by retokromer** [comment]
Oh, just pass the parameter with an existing tag, instead of create an empty one for this.

File: index.html [hunk 1]
```diff
@@ -58,15 +58,16 @@
...
       </div>
     </div>
   </div>
   <!-- ends WAV to MP3 -->
 
   <!-- Transcode to ProRes -->
-  <span data-toggle="modal" data-target=".to_prores"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="This will transcode to deinterlaced Apple ProRes LT">Transcode to ProRes</button></span>
-  <div class="modal fade to_prores" tabindex="-1" role="dialog">
+  <span data-toggle="modal" data-target="#to_prores"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="This will transcode to deinterlaced Apple ProRes LT">Transcode to ProRes</button></span>
+  <div id="to_prores" class="modal fade" tabindex="-1" role="dialog">
```
Comment (retokromer) [inline index.html:70]: The _Transcode to ProRes_ section does **not** contain the
`<p class="link"></p>`
part. Yet it seems to me to work fine. Is only one occurrence needed?

**Comment by kieranjol** [comment]
Well the link works but a 'link to this command' does not appear at the bottom of the recipe: https://amiaopensource.github.io/ffmprovisr/#to_prores

**Comment by retokromer** [comment]
I see, thank you! When back on a «normal» computer, I will try into h3, otherwise I will add an empty p.

**Comment by retokromer** [comment]
Finally added another `<p class="link"></p>` (https://github.com/amiaopensource/ffmprovisr/pull/65).

**Comment by retokromer** [comment]
Hello @kieranjol, by chance do you know how we can make the links live (I mean clickable)? If not, I will delve into the code you used.

**Comment by kieranjol** [comment]
Sorry Reto,i don't know, but I did wish that they were clickable..

**Comment by retokromer** [comment]
No problem, I will check. (My question was just to avoid to do twice a job :-)

File: js/js.js [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+  $(function () {
+    $('[data-toggle="tooltip"]').tooltip()
+  });
+
+  // open modal window if a hash is found in URL
+  if(window.location.hash) {
+    $(window.location.hash).modal('show');
+    // add direct link to modal window
+    $(".link").empty();
+    $(".link").append("<small>Link to this command: "+window.location.href+"</small>");
```
Comment (retokromer) [inline js/js.js:12]: I presume something like
`$(".link").append("<small>Link to this command: <a href="+window.location.href+">"+window.location.href+"</a></small>");`
should work. In order to respect the HTML5 standard, also additional " around the path should be added. I have to check how to do this in that language I never use.

File: js/js.js [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+  }
+
+  // add hash to URL when modal is opened
+  $('span[data-toggle="modal"]').on("click", function(){
+    window.location.hash = $(this).attr("data-target");
+    // add direct link to modal window
+    $(".link").empty();
+    $(".link").append("<small>Link to this command: "+window.location.href+"</small>");
```
Comment (retokromer) [inline js/js.js:20]: same here

File: js/js.js (hunk 2 above)
Comment (retokromer) [inline js/js.js:12]: Possibly:
`$(".link").append("<small>Link to this command: <a href=&quot;"+window.location.href+"&quot;>"+window.location.href+"</a></small>");`

**Comment by retokromer** [comment]
I made a test: https://github.com/amiaopensource/ffmprovisr/pull/68

**Pull request merged**